src/mp4/ilst/mod.rs

```diff
@@ -84,6 +84,11 @@ pub struct Ilst {
 
 impl Ilst {
 	/// Returns all of the tag's atoms
+	///
+	/// # Deprecated
+	///
+	/// Use one of the [`IntoIterator`] implementations for accessing all atoms.
+	#[deprecated]
 	pub fn atoms(&self) -> &[Atom<'static>] {
 		&self.atoms
 	}
```

Comment:
```suggestion
```
No need to mark this as deprecated, it can just be removed.

Reply:
Amended and force pushed.